less/events.less

```diff
@@ -0,0 +1,103 @@
+/* @CHARSET "UTF-8"; */
+/*!
+ * Copyright (c) 2019 Eclipse Foundation, Inc.
+ * 
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ * 
+ * Contributors:
+ *   Christopher Guindon <chris.guindon@eclipse-foundation.org>
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+*/
+
+// @import '~eclipsefdn-solstice-assets/less/quicksilver/styles.less';
+// @import '_variables.less';
+// @import (less) "~owl.carousel/dist/assets/owl.carousel.min.css";
+// @import (less) "~owl.carousel/dist/assets/owl.theme.default.min.css";
+
+// @image-url: "/static/images/eventsImages";
+
+.eventCard {
```

Comment:
As per our naming convention, we try to create classes like this:
.event-card

"event-" would be a prefix that we use on all of our classes for the Events website.